configure.ac

```diff
@@ -311,7 +311,7 @@ If your host is 64 bits, you can try with './configure CC="gcc -m64"' \
 
 AS_CASE([$host],
   [*-pc-windows],
-    [CC=cl
+    [AS_CASE([$CC], [cl|cl.exe|clang-cl|clang-cl.exe], [], [CC=cl])
```

Comment:
Is there a reason not to set `CC` only if it is empty?
(When I explicitly set some variable, I’m pretty annoyed if that’s overridden)

Reply:
IIRC the problem is that Autoconf sets `CC` automatically before this line. We have no way of knowing whether the user sets `CC` in `./configure CC=xxx` or if Autoconf discovered the compiler; and Autoconf has a tendency of picking gcc instead of cl or clang-cl. I'll double-check.

Reply:
I was mistaken! It's possible to check if `CC` is empty. Good catch, thanks.